x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/parameters_definition.tsx

```diff
@@ -460,6 +490,7 @@ export const PARAMETERS_DEFINITION = {
         defaultMessage: 'Value must be greater than 0.',
       }),
     } as FieldConfig,
+    schema: Joi.string(),
```

Comment:
I think this should be `Joi.number()`, right? I tried loading up the [metricbeat mappings](https://gist.githubusercontent.com/cjcenizal/09837195a7b1f2ba1447aa4d67859497/raw/c8a9e541d3797adea2a6e89327f860a8263be5ab/metricbeat_index_template) and the editor complained about this until I changed it to `Joi.number()`.

Reply:
Great catch! I also had to add a deserializer on the parameter config as ES allows a string to be defined, but our form input requires an integer.

This is OK for ES

<img width="476" alt="Screen Shot 2019-12-30 at 14 26 36" src="https://user-images.githubusercontent.com/2854616/71574887-b4bd5700-2b10-11ea-9cc5-e1e8001fe586.png">